tests/fixtures/app/typescript/.eslintrc.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
+  },
+  plugins: ['ember', '@typescript-eslint'],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended',
+    'plugin:prettier/recommended',
+  ],
+  env: {
+    browser: true,
+  },
+  rules: {},
+  overrides: [
+    // ts files
```

Comment:
Putting the TS `extends` into an override for *only* .ts files is not something the RFC explicitly mandated, but it is what I have been doing everywhere. Otherwise, when putting this into the global `extends`, you would get false positives from TS rules when ESLint runs on plain .js files.

If you suggest a different solution, please tell me!